integrated_channels/cornerstone/transmitters/content_metadata.py

```diff
@@ -16,12 +16,27 @@ def __init__(self, enterprise_configuration, client=CornerstoneAPIClient):
             client=client
         )
 
-    def _transmit_action(self, content_metadata_item_map, client_method, action_name):
+    def transmit(self, create_payload, update_payload, delete_payload, **kwargs):
```

Comment:
i dont personally think we should handle the cache via a kwarg here. i'll huddle and suggest an alternative

Reply:
if we use a kwarg here to skip clearing of these marked records, lets make sure it also causes us to skip updating their timestamps, api reponses, and error messages

Reply:
you can probably drop the `**kwargs`